health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -0,0 +1,100 @@
+        }
+
+    )
+
+
+
+
+
+
+    // Use useEffect to update Files only when data changes
+    useEffect(() => {
+        if (data && data["ResourceDetails"]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)